lib/linguist/languages.yml

```diff
@@ -4207,6 +4207,14 @@ Python traceback:
   tm_scope: text.python.traceback
   ace_mode: text
   language_id: 304
+QL:
+  type: programming
+  extensions:
+  - ".ql"
+  - ".qll"
```

Comment:
We'll need a search query for this extension, as well as a sample file.

Reply:
@pchaigno thank you for catching this. My apologies for overlooking that. I've updated the PR body to include a `.qll` search along with `.qll` samples and their sources (the samples provided are licensed under Apache 2.0).

Reply:
Thanks!